ERCS/erc-7902.md

```diff
@@ -0,0 +1,200 @@
+---
+eip: 7902
+title: Wallet Capabilities for Account Abstraction
+description: New Wallet Capabilities providing a way for dApps and Account Abstraction wallets to communicate
+author: Yoav Weiss (@yoavw), Alex Forshtat (@forshtat), Dror Tirosh (@drortirosh), Shahaf Nacson (@shahafn)
+discussions-to: https://ethereum-magicians.org/t/erc-wallet-capabilities-for-account-abstraction/23122
+status: Draft
+type: Standards Track
+category: ERC
+created: 2025-03-01
+requires: 5792, 7702
+---
+
+## Abstract
+
+[EIP-5792](./eip-5792) defines a baseline JSON-RPC API for a communication between wallets and dapps.
+With EIP-5792, dapps and wallets can communicate advanced features using "capabilities" - extensions
+to the base protocol that must be defined in separate documents.
+
+This proposal defines a set of "capabilities" the wallets may want to implement in order to provide a comprehensive support for Account Abstraction.
```

Comment:
```suggestion
This proposal defines a set of "capabilities" the wallets may want to implement in order to provide a comprehensive support for Account Abstraction (AA).
```

Probably a well-known abbreviation at this point, but just in case...